app/javascript/controllers/chat_controller.js

```diff
@@ -29,7 +29,28 @@ export default class extends ApplicationController {
     if (this.element.value.length === 0) {
       return;
     }
-    this.stimulate('ChatReflex#prompt')
+
+    const xhr = new XMLHttpRequest();
```

Comment:
When would this fire off?

Reply:
a better solution is probably to put the check in the ApplicationReflex, similar to how it works in ApplicationController

Reply:
would be executed after session verification its done.

Yes let me check how to do that